src/cmd/vm.rs

```diff
@@ -236,23 +244,24 @@ pub struct ArgsStart {
 }
 
 fn run_start(args: &ArgsStart) -> Result<()> {
-    if args.acloud {
-        run_acloudw(args)?;
+    let port = if args.acloud {
+        run_acloudw(args)?
     } else {
         run_betty_start(args)?;
-        let info = register_dut("127.0.0.1:9222")?;
+        "9222".to_string()
+    };
 
-        println!(
-            "You can connect the VM instance with `lium dut shell --dut {:?}`.",
-            info.id()
-        );
-        println!("To push an Android build a betty VM, run `lium arc flash`.");
-    }
+    let info = register_dut(&format!("127.0.0.1:{port}"))?;
```

Comment:
Maybe adding a comment why 127.0.0.1 is here is helpful for the users!

Reply:
Added a comment, thank you!